partiql-types/src/main/kotlin/org/partiql/errors/ErrorCode.kt

```diff
@@ -365,6 +362,12 @@ enum class ErrorCode(
                 "got: ${errorContext?.get(Property.ACTUAL_ARGUMENT_TYPES) ?: UNKNOWN}"
     },
 
+    /**
+     * NOTE: This is unused and may be removed!
+     *
+     * To remove the dep on ExprValue, we introduce the EXPECTED_ARGUMENT_TYPES argument
+     *  Property.EXPECTED_ARGUMENT_TYPES -> ExprValueType.values().filter { it.isText }
+     */
     EVALUATOR_CONCAT_FAILED_DUE_TO_INCOMPATIBLE_TYPE(
         ErrorCategory.EVALUATOR,
         LOCATION + setOf(Property.ACTUAL_ARGUMENT_TYPES),
```

Comment:
```suggestion
        LOCATION + setOf(Property.ACTUAL_ARGUMENT_TYPES, Property.ACTUAL_ARGUMENT_TYPES,),
```

I believe you need to add this.

Reply:
Do you mind clarifying why? Did you mean to put the same thing twice?

Reply:
Oh, sorry: I meant to add `EXPECTED_ARGUMENT_TYPES` instead.

And it loosely tells the end user what properties are available. So, since your expect `EXPECTED_ARGUMENT_TYPES` to be in the property value map, it can be accessed by the end-user.